streaming/base/storage/upload.py

```diff
@@ -73,6 +74,7 @@ def get(cls,
                 shard file or remove it after uploading. Defaults to ``False``.
             progress_bar (bool): Display TQDM progress bars for uploading output dataset files to
                 a remote location. Default to ``False``.
+            exist_ok (bool): When exist_ok = False, raise error if out already exists and has contents. Defaults to ``False``.
```

Comment:
What `out` value are you referring to ? Out can local, remote, or both. Can you please clarify that in the method args comment !

Reply:
Done.